cmd/juju/commands/ssh_container.go

```diff
@@ -450,6 +451,12 @@ func (c *sshContainer) getCloudSpec() (cloudspec.CloudSpec, error) {
 	}
 	c.modelName = mInfo.Result.Name
 
```

Comment:
Code below line 443 are legacy. We should find the controller name before line 439(probably just do it in `initRun` or `resolveTarget`).